lib/iris/fileformats/netcdf.py

```diff
@@ -1043,7 +1043,7 @@ def write(
             dtype(i.e. 'i2', 'short', 'u4') or a dict of packing parameters as
             described below. This provides support for netCDF data packing as
             described in
-            http://www.unidata.ucar.edu/software/netcdf/docs/BestPractices.html#bp_Packed-Data-Values
+            http://www.unidata.ucar.edu/software/netcdf/documentation/NUG/best_practices.html#bp_Packed-Data_Values
```

Comment:
A bit of an overlap with the changes in here: https://github.com/SciTools/iris/pull/4132/files, but we can rebase after we've merged

Reply:
I think #4132 will be redundant once all tests are passing.